wpilibj/src/main/java/edu/wpi/first/wpilibj/Compressor.java

```diff
@@ -33,30 +33,16 @@
    */
   public Compressor(int module, PneumaticsModuleType moduleType) {
     m_module = PneumaticsBase.getForType(module, moduleType);
-    boolean allocatedCompressor = false;
-    boolean successfulCompletion = false;
 
-    try {
-      if (!m_module.reserveCompressor()) {
-        throw new AllocationException("Compressor already allocated");
-      }
-
-      allocatedCompressor = true;
-
-      m_module.enableCompressorDigital();
```

Comment:
There was a warning about using try-with-resources instead of calling close() in finally, but I rewrote the code instead.